src/scaffoldmaker/meshtypes/meshtype_3d_brainstem.py

```diff
@@ -278,10 +637,10 @@ def defineFaceAnnotations(cls, region, options, annotationGroups):
         # external regions
         namelist = ['midbrain', 'pons', 'medulla oblongata']
         for subregion in namelist:
-            subGroup = AnnotationGroup(region, get_brainstem_annotation_term(subregion))
+            subGroup = AnnotationGroup(region, get_brainstem_term(subregion))
             issub = subGroup.getFieldElementGroup(mesh2d)
             is_subface = fm.createFieldOr(fm.createFieldAnd(issub, is_exterior_face_xi1), fm.createFieldAnd(issub, is_exterior_face_xi3))
-            subFaceGroup = findOrCreateAnnotationGroupForTerm(annotationGroups, region, (subregion+'_exterior', None))
+            subFaceGroup = findOrCreateAnnotationGroupForTerm(annotationGroups, region, (subregion + '_exterior', None))
             subFaceGroup.getMeshGroup(mesh2d).addElementsConditional(is_subface)
 
```

Comment:
The exterior annotation groups need to be defined in annotation/brainstem_terms; the term IDs need to be changed from None to "None" -- that is the cause of the crash on clicking Done. ILX terms need to be requested as well.
Remove underscores in term names.